proto/buf/alpha/convert/v1alpha1/convert.proto

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.convert.v1alpha1;
+
+// ModuleInfo contains information about a module that a serialized message belongs to.
+//
+// This definition is equivalent to buf.alpha.image.v1.ModuleInfo, but it's
+// purposefully redefined here to prevent import cycles in the generated code.
+message ModuleInfo {
```

Comment:
Why is this needed? This shouldn't be a separate type in a `convert` package, we should use the existing types